outsource/outsource.py

```diff
@@ -312,6 +311,44 @@ def _generate_tc(self):
     def _generate_rc(self):
         return ReplicaCatalog()
 
+    def make_tarballs(self):
+        """Make tarballs of Ax-based packages if they are in editable user-
+        installed mode."""
+        tarballs = []
+        tarball_paths = []
+        for package_name in ["strax", "straxen", "cutax"]:
+            _tarball = Tarball(self.generated_dir, package_name)
+            if not Tarball.get_installed_git_repo(package_name):
+                # Packages should not be non-editable user-installed
+                if Tarball.is_user_installed(package_name):
```

Comment:
Why we should forbid non-editable user-installed?

Reply:
Because a bare module does not contain how it should be installed. i.e. there is not `setup.py` or `pyproject.toml` in `.local/lib/python3.9/site-packages`.